build.gradle

```diff
@@ -135,8 +135,10 @@ dependencies {
     providedRuntime("org.springframework.boot:spring-boot-starter-tomcat")
 
     //Injectable Message Library and its Implementation
-    compile 'com.github.Ericsson:eiffel-remrem-shared:0.1.4'
-    compile ('com.github.Ericsson:eiffel-remrem-semantics:0.1.7')
+    //compile 'com.github.Ericsson:eiffel-remrem-shared:0.1.5'
+    compile files ('/home/evasiba/git/eiffel-remrem-shared/build/libs/remrem-shared-0.1.5.jar')
```

Comment:
I'll remove them but we will not merge this until we have released new version of eiffel-remrem-shared, and eiffel-remrem-semantic